src/main/java/com/pingcap/tikv/expression/scalar/LeftShift.java

```diff
@@ -39,4 +39,9 @@ public String getName() {
   public DataType getType() {
     throw new UnsupportedOperationException();
   }
+
+  @Override
+  ScalarFuncSig getSignature() {
+    throw new UnsupportedOperationException();
```

Comment:
Do we indeed have this operator in new DAG mode? If no, remove this class entirely.

Reply:
We don't need this. Class removed.